test/unit/parser.test.js

```diff
@@ -1346,6 +1346,18 @@ describe('Parser', () => {
       const filter = parser.parse().value;
       assert.isTrue(filter.match(src));
     });
+
+    it('should match true on isRegex', () => {
+      const parser = new Parser('/corge like "[a-z]{1}"');
+      const filter = parser.parse().value;
+      assert.isTrue(filter.match(src, true));
+    });
+
+    it('should match false on isRegex', () => {
+      const parser = new Parser('/foo like "\\w"');
```

Comment:
Check that this test should really return `false`, as it seems to be a valid regexp.

Reply:
It is because /foo is not a string. I change it